mintpy/view.py

```diff
@@ -422,36 +425,37 @@ def plot_slice(ax, data, metadata, inps=None):
             vprint('plot in Lat/Lon coordinate')
             vprint('map projection: '+inps.map_projection)
```

Comment:
We should use `map_projection` argument to specify the project, as a choice with default for `PlateCarree` in string. Then convert it to cartopy projection object, for example as below:

```
PROJECTION_NAME2OBJ = {
    'PlateCarree': ccrs.PlateCarree(),
    ...
}

inps.map_projection = PROJECTION_NAME2OBJ(inps.map_projection)
```

Reply:
Yes. I agree with this approach.

Reply:
Is there a specific place I should place the `PROJECTION_NAME2OBJ` dict?

Reply:
This is a global variable, so we could put it on the top of view.py.